jbehave-support-core/src/main/java/org/jbehavesupport/core/rest/RestServiceSteps.java

```diff
@@ -80,6 +79,16 @@ public void verifyResponse(String application, String status, ExamplesTable data
         resolveHandler(application).verifyResponse(status, data);
     }
 
+    @Then("response from [$application] REST API is successful")
+    public void verifySuccessfulResponse(String application) {
+        resolveHandler(application).verifySuccessfulResponse(null);
+    }
+
+    @Then("response from [$application] REST API is successful and values match: $data")
+    public void verifySuccessfulResponse(String application, ExamplesTable data) {
+        resolveHandler(application).verifySuccessfulResponse(data);
```

Comment:
Those new steps should be documented (in javadoc and wiki)

Reply:
Docu added.